sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -4300,7 +4305,7 @@ SIREPO.app.directive('particle3d', function(appState, panelState, requestSender,
                     zAxisRight = Math.min(sceneZEnds[1][0], clippedZEnds[1][0]);
                     zAxisBottom = zAxisRight == sceneZEnds[1][0] ? sceneZEnds[1][1] : clippedZEnds[1][1];
                     zAxisProjLen = plotUtilities.dist([zAxisLeft, zAxisTop], [zAxisRight, zAxisBottom]);
-                    var tanTheta = (sceneZEnds[0][1] - sceneZEnds[1][1]) / (sceneZEnds[0][0] - sceneZEnds[1][0]);
+                    tanTheta = (sceneZEnds[0][1] - sceneZEnds[1][1]) / (sceneZEnds[0][0] - sceneZEnds[1][0]);
```

Comment:
this seems to be more [impedance mismatches](https://github.com/radiasoft/pykern/wiki/DesignHints#impedance-matches-data-structures-align-with-algorithms) and [differentiate differences](https://github.com/radiasoft/pykern/wiki/DesignHints#differentiate-differences).

Reply:
Yes, all that old stuff is still in place while I fix it up...